src/common/image.c

```diff
@@ -46,6 +46,43 @@
 #endif
 #include <glib/gstdio.h>
 
+int64_t max_image_position()
+{
+  sqlite3_stmt *stmt = NULL;
+
+  // get last position
+  int64_t max_position = 0;
+
+  gchar *max_position_query = g_strdup("SELECT MAX(position) FROM main.images");
+  DT_DEBUG_SQLITE3_PREPARE_V2(dt_database_get(darktable.db), max_position_query, -1, &stmt, NULL);
+
+  if (sqlite3_step(stmt) == SQLITE_ROW)
+  {
+    max_position = sqlite3_column_int64(stmt, 0);
+  }
+
+  sqlite3_finalize(stmt);
+  g_free(max_position_query);
+  return max_position;
+}
+
+int64_t createNextImagePosition()
```

Comment:
Make it static, too.

And please, no camelCase.

Reply:
Switching programming languages :-) ..... Done